lib/phoenix/live_dashboard/live/os_mon_live.ex

```diff
@@ -0,0 +1,264 @@
+defmodule Phoenix.LiveDashboard.OSMonLive do
+  use Phoenix.LiveDashboard.Web, :live_view
+
+  alias Phoenix.LiveDashboard.{
+    SystemInfo,
+    BarComponent,
+    ColorBarComponent,
+    ColorBarLegendComponent
+  }
+
+  @temporary_assigns [system_info: nil, system_usage: nil]
+
+  @cpu_usage_sections [
+    {:kernel, "Kernel", "purple"},
+    {:user, "User", "blue"},
+    {:nice_user, "User nice", "green"},
+    {:soft_irq, "Soft IRQ", "orange"},
+    {:hard_irq, "Hard IRQ", "yellow"},
+    {:steal, "Steal", "purple"},
+    {:wait, "Wait", "orange"},
+    {:idle, "Idle", "dark-gray"}
+  ]
+  @memory_usage_sections [
+    {:in_use_memory, "In use", "purple"},
+    {:buffered_memory, "Buffered", "green"},
+    {:cached_memory, "Cached", "orange"},
+    {:free_memory, "Free", "dark-gray"}
+  ]
```

Comment:
Unfortunately we may have more or less entries than this. They are OS specific.

Reply:
I think for memory data, it is best to show them as individual entries, like we do for Input, Output, etc. Since the different values relate to each other in many different ways.